### ISSUE TITLE
Remove lodash

### ISSUE THREAD

**Issue by lteacher** [lteacher/lambda-chain#5]
Love lodash, but its unnecessary in this project and increases size for nothing. I should remove it.

**Comment by nab911** [comment]
I will go ahead and tackle this one as well for hacktoberfest.

**Comment by lteacher** [comment]
Sure you are welcome to do it. I didn't label it cause its a little more annoying than the other issue so didn't want to troll anyone.

Lodash is used in a few ways which might be annoying to fix up because originally I wrote this stuff in ES6 and then webpacked / 'bundled' it up with the serverless babel plugin, but it meant in a few places I had used spread on args and to replace that I had some dramas with binding and ended up currying some functions which also resolved that binding issue.

There are a few other checks for object types as well.

**Comment by nab911** [comment]
Yeah, looking at it now, I believe it's better to leave it in there. Lodash is such a popular package that I would expect most other projects using your project would have lodash anyways so it won't increase the size.
